.github/workflows/backend.yml

```diff
@@ -4,63 +4,90 @@ on:
   pull_request:
 
 jobs:
+  lint-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+
+      - name: npm install
+        run: npm install prettier prettier-plugin-ini
```

Comment:
in a follow up, we might want to have it in a `package.json` so it does not break on breaking version bumps.